rollup-tests.config.js

```diff
@@ -52,7 +52,20 @@ export default {
         [
           'babel-plugin-istanbul',
           {
-            exclude: ['**/test/**/*.js', '**/tests/**/*.js', '**/test-util/**'],
+            exclude: [
+              '**/node_modules/**',
+              '**/test/**/*.js',
+              '**/tests/**/*.js',
+              '**/test-util/**',
+            ],
+
+            // These two configuration options needed to match the values that
+            // vitest sets internally, so that they pick the coverage
+            // instrumentation generated by babel-plugin-istanbul instead of
+            // trying to generate their own.
+            // See https://github.com/vitest-dev/vitest/discussions/7841#discussioncomment-12855608
+            coverageGlobalScope: 'globalThis',
+            coverageVariable: '__VITEST_COVERAGE__',
```

Comment:
We could possibly move this workaround to `@hypothesis/frontend-testing` in future so we only have to document it in one place.

Reply:
Yep, that makes sense. I need to spend some time thinking what's the right way to do that.

Reply:
Maybe a `vitestBabelPluginIstanbulOptions(exclude)` function would be a good option.